Scripts/extract_release_notes.rb

```diff
@@ -0,0 +1,80 @@
+# Parses the release notes file to extract the current version information and
+# puts it to STDOUT.
+#
+# To update the release notes for localization:
+#
+# ruby ./this_script >| Simplenote/Resources/release_notes.txt
+#
+# To generate the App Store Connect release message:
+#
+# ruby ./this_script | pbcopy
+#
+# To generate the GitHub and App Center release message:
+#
+# ruby ./this_script -k | pbcopy
+
+GITHUB_URL = 'https://github.com/Automattic/simplenote-macos'
+
+RELEASE_NOTES_FILE = 'RELEASE-NOTES.txt'
+NOTES = File.read(RELEASE_NOTES_FILE)
+lines = NOTES.lines
+
+def replace_pr_number_with_markdown_link(string)
+  string.gsub(/\#\d*$/) do |pr_number|
```

Comment:
Same feedback as in https://github.com/Automattic/simplenote-ios/pull/1202#pullrequestreview-608589162